src/base/numbers.c

```diff
@@ -74,3 +74,42 @@ int _getHexString(char *buffer, unsigned long hexnumber, bool use_capital) {
     }
     return size;
 }
+
+// Unsigned
+int itoa(char *buffer, unsigned long number, int base, bool use_capital) {
+	if (base < 1 || base > 36) return 0;
```

Comment:
Hey! I'm one of the other contributors of dreamos64, the code looks good but I have some feedback about the style. Namely putting the `return` on a separate line so it's clearer:
```c
if (base < 1 || base > 36)
    return 0;
```